### DISCUSSION TITLE
articles/siddhartha

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#141] [Comments]
https://hutusi.com/articles/siddhartha 

 最近读了本小说，赫尔曼·黑塞的《悉达多》1。书名看上去像是在写释迦牟尼的故事，实则不然。作者并未写明他是佛祖，而是借用了释迦牟尼的故事，但将佛祖俗世的名字“乔达摩·悉达多”分成了书中的两个人物：乔达摩和悉达多。但我更认可豆瓣中一位网友的评论解读2： 《悉达多》(Siddhartha), 又被译为《流浪者之歌》，德国小说家赫尔曼·黑塞于1922年所著。 ↩ 对赫尔曼.黑塞的《悉达多》的解读: https://book.douban.com/review/6390024/ ↩

**Comment by ColarXu** [comment]
我会思考，我会等待，我会斋戒。
